### PR TITLE
query and update animals table

### PR DESCRIPTION
**Pull request by ha-manel**
### In this PR, I updated the `animals` table and added queries for it.
roll
pokemondigimon
delete
savepoint
qsts

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -10,3 +10,5 @@ CREATE TABLE animals (
     neutered BOOLEAN,
     weight_kg DECIMAL
 );
+
+ALTER TABLE animals ADD species varchar;
```
Comment (keneogu) [inline schema.sql]: Add the varchar limit

File: data.sql [hunk 2]
```diff
@@ -8,3 +8,11 @@ INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Gabumon', 'Nov 15, 2018', 2, true, 8);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Pikachu', 'Jan 7, 2021', 1, false, 15.04);
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Devimon', 'May 12, 2017', 5, true, 11);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Charmander', 'Feb 8, 2020', 0, false, 11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Plantmon', 'Nov 15, 2021', 2, true, 5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Squirtle', 'Apr 2, 1993', 3, false, 12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) VALUES ('Angemon', 'Jun 12, 2005', 1, true, 45);
```
Comment (keneogu) [inline data.sql]: Angemon weight should be -45.0

File: queries.sql [hunk 3]
```diff
@@ -3,10 +3,44 @@
...
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT name, species FROM animals;
+ROLLBACK;
+SETECT name, species FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
```
Comment (keneogu) [inline queries.sql]: before the COMMIT you have to verify that the change was made by adding `SELECT species from animals`

**Review by keneogu — CHANGES_REQUESTED** [review]
Hi @ha-manel,

Good job so far!

- All linters are passing
- You are following GitHub flow
- Good commit history

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql [hunk 4]
```diff
@@ -15,13 +15,13 @@ BEGIN;
...
 ROLLBACK;
-SETECT name, species FROM animals;
+SELECT name, species FROM animals;
 
 BEGIN;
 UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
 UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
-COMMIT;
 SELECT name, species FROM animals;
+COMMIT;
```
Comment (DeliceLydia) [inline queries.sql:24]: ## Optional
- [ ] I suggest that you do as follows:
 ```
SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit

```

**Review by DeliceLydia — APPROVED** [review]
## Hi @ha-manel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**